velox/docs/functions/presto/string.rst

```diff
@@ -160,6 +160,11 @@ String Functions
 
     Converts ``string`` to uppercase.
 
+.. function:: levenshtein_distance(string_1, string_2) -> integer
```

Comment:
Would you double check the return type. It looks like the code is using bigint, not integer.

Reply:
yes, it should be bigint. The case will be one string is empty and the other one's length doesn't fit in int.

Reply:
I don't think we can have strings that long, but we must match Presto:

```
presto:di> show functions like 'levenshtein_distance'
        -> ;
       Function       | Return Type |     Argument Types     | Function Type | Deterministic |                    Description                    | Variable Arity | Built In | Temporary | Language
----------------------+-------------+------------------------+---------------+---------------+---------------------------------------------------+----------------+----------+-----------+----------
 levenshtein_distance | bigint      | varchar(x), varchar(y) | scalar        | true          | computes Levenshtein distance between two strings | false          | true     | false     |
(1 row)
```

Reply:
Definitely, have made that change. From the code perspective, since we have a combination input size check to 1'000'000 will be safe to int.  The only case for it hitting int limit is string1 length is very large and string2 is empty, which is not possible since we cannot have string with such a big length.